.travis.yml

```diff
@@ -2,22 +2,18 @@ services:
 - docker
 
 before_script:
-  - docker login -u $DOCKER_HUB_USER -p $DOCKER_HUB_PASSWORD   #login to dockerhub
-  - curl https://cli-assets.heroku.com/install.sh | sh  #install heroku
-  - docker login --username=$HEROKU_LOGIN --password=$HEROKU_API_KEY registry.heroku.com 
+  - docker login -u $DOCKER_HUB_USER -p $DOCKER_HUB_PASSWORD   #login to dockerhub  
 
 script:
   - docker build --target test --tag my-test-image .
   - docker run my-test-image test_items.py
   - docker run -e MONGO_DB_URL=$MONGO_DB_URL  my-test-image test_client.py
   - docker build --target production --tag $DOCKER_HUB_USER/todo-app:latest .  
-  - docker push $DOCKER_HUB_USER/todo-app:latest
-  - docker tag $DOCKER_HUB_USER/todo-app:latest registry.heroku.com/$HEROKU_APP/web
-  - docker push registry.heroku.com/$HEROKU_APP/web
+  - docker push $DOCKER_HUB_USER/todo-app:latest 
```

Comment:
It might be worth moving this line to the deploy section as well, I'd normally expect the `latest` tag to track master. Fine either way though